src/GraphQL.Harness.Tests/GraphQL.Harness.Tests.csproj

```diff
@@ -11,6 +11,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\GraphQL.Harness\GraphQL.Harness.csproj" />
```

Comment:
GraphQL.Harness contains `<ProjectReference Include="..\GraphQL.NewtonsoftJson\GraphQL.NewtonsoftJson.csproj" />`. Need to be removed either from here or from there.

Reply:
Removed these refs. STJson one is now inherited from harness project and is the only one tested there now.